### DISCUSSION TITLE
expo cli in docker container

### DISCUSSION THREAD

**Discussion by 343max** [ente-io/ente#3609] [Q&A]
### Description

I'm running ente inside a docker contiainer using the provided documentation. Now I want to make my own user the admin user and in order to do so I need to find my own user id. According to the documentation this should be done using the `ente account list` command. But I can't figure out how to run this command inside the docker container.

How do I run `ente account list` when the server runs inside a docker container?

### What product are you using?

Ente Photos

### What platform are you using?

Server

**Comment by mnvr** [comment]
You don't need to run the `ente account list` command within the server's docker container. This is a separate command. You need to configure the CLI (see the documentation for hints) to connect to your server instance, then you can run it from anywhere.

Also note - the first user created within the server automatically is granted admin privileges. This facility is available in the latest server docker image (you might be using an old one if you didn't notice this).
